syft/frameworks/torch/torch_attributes.py

```diff
@@ -1,6 +1,9 @@
 from typing import Union
 from typing import Callable
 from typing import Any
+
+
+# @trask - my IDE can't find this - does anyone else's see it
```

Comment:
Which one can't it find?
Also these type of comments belong in the slack or as an issue rather than a comment in the code

Reply:
done